### DISCUSSION TITLE
Transaction Stuck At Waiting for Wormhole Network Consensus

### DISCUSSION THREAD

**Discussion by amol3335** [wormhole-foundation/wormhole#513] [Problems / bugs] [answered]
https://explorer.solana.com/tx/5h9Ww2du6XHBTMBRGz7zwYhTp6BVPECcwzDtSWyeJrboH32KDYnEbWRSSc2acBayov5souo842myMu3gRzSf5fHm
Transaction Stuck.

**Comment by evan-gray** [comment]
Looks like this transaction was missed by the guardians and should be picked up after their next update. (I think someone replied similarly on Discord)

**Comment by amol3335** [comment, marked as answer]
> Looks like this transaction was missed by the guardians and should be picked up after their next update. (I think someone replied similarly on Discord)

When is this Update scheduled??

**Reply by leoluk** [reply]
It happened already, actually! Please try redeeming your transaction on https://www.wormholebridge.com, it should work now.
